lib/ash/resource/change/builtins.ex

```diff
@@ -286,6 +286,47 @@ defmodule Ash.Resource.Change.Builtins do
     {Ash.Resource.Change.Select, target: value, ensure?: true}
   end
 
+  @doc """
+  Cascade this resource's destroy action to a related resource's destroy action.
+
+  Adds an after-action hook that explicitly calls destroy on any records related
+  via the named relationship.  It will optimise for bulk destroys where
+  possible.
+
+  > #### Beware database constraints {: .warning}
+  >
+  > Think carefully before using this change with data layers which enforce
+  > referential integrity (ie PostgreSQL and SQLite) and you may need to defer
+  > constraints for the relationship in question.
+  >
+  > See also:
+  >   1. [`postgres.references.reference.deferrable` DSL](`e:ash_postgres:dsl-ashpostgres-datalayer.html#postgres-references-reference-deferrable`)
```

Comment:
What is the `e:` syntax?

Reply:
https://hexdocs.pm/ex_doc/readme.html#auto-linking

turns out it emits a warning if the package isn't in the deps, so I've converted it to a normal link.